django/core/checks/model_checks.py

```diff
@@ -8,12 +10,14 @@
 
 @register(Tags.models)
 def check_all_models(app_configs=None, **kwargs):
+    db_table_attr_map = defaultdict(list)
     errors = []
     if app_configs is None:
         models = apps.get_models()
     else:
         models = chain.from_iterable(app_config.get_models() for app_config in app_configs)
     for model in models:
+        db_table_attr_map[model._meta.db_table].append(model.__name__)
```

Comment:
Use [`model._meta.label`](https://github.com/django/django/blob/f091ea35150d95fc6732bbf0c27b971dd445509a/django/db/models/options.py#L134-L136) instead of `__name__` as `label` is namespaced by `app_label` which will make it easier to identify collisions between apps. Also you only want to consider [managed models](https://docs.djangoproject.com/en/2.1/topics/db/models/#differences-between-proxy-inheritance-and-unmanaged-models) here so you don't want to add values to `db_table_attr_map` in the case of `not model._meta.managed`.